src/control/server/config/server_test.go

```diff
@@ -129,8 +129,14 @@ func mockConfigFromFile(t *testing.T, path string) (*Server, error) {
 	t.Helper()
 	c := DefaultServer()
 	c.Path = path
+	err := c.Load()
+	if err == nil {
+		for i := 0; i < len(c.Engines); i++ {
+			c.Engines[i] = c.Engines[i].WithProperEnvVarForPMDK()
+		}
+	}
```

Comment:
I wonder if there'd be some value in making these modifications in Load() instead of Validate().

Reply:
I think @kjacque is right, Validate() should as much as possible be a read-only operation, Load() is probably a better place. Also then we shouldn't see the output generated from config generate (auto.go) API which seems cleaner to me.